localization/english/wc_regiment_l_english.yml

```diff
@@ -14,7 +14,155 @@
+ wyvern_rider_flavor:0 "#F Together with our new allies, the wyverns, we have created a unit of skilled aerial combatants, the Wind Riders. At long last, the skies are within our grasp."
+
+ ##Early Medival
+ baradian_marine_flavor:0 "#F The elite of our military, Baradian Marines know no fear, only glorious victory. In peacetime they guard the prison of Tol Barad, but it is in war that they truly shine."
+ bearpaws_flavor:0 "#F Among the pandaren, few are as physically powerful as the Bearpaws. Clad in bamboo armor and possessing formidable skill in martial arts, Bearpaws are the first line of defense for the Pandaren."
+ blackrock_legionnaire_flavor:0 "#F The best of the Blackrock Clan, the Legionnaires defy expectations as an orcish force with iron discipline. They conquer all in their path, showing no mercy."
+ chaosborn_flavor:0 "#F The chaosborn are powerful eredar sworn to Sargeras. They are dangerous opponents, skilled with both the sword and fel magic."
+ chimaera_flavor:0 "#F The fearsome two-headed Chimaeras circle the dark forests of northern Kalimdor, fighting with their terrible breath against all who desecrate the purity of their homeland."
+ cloud_serpents_flavor:0 "#F Long have we held close bonds with our friends in the sky. Now, our bond is closer than ever, as we ride the Cloud Serpents to new heights."
+ faerie_dragon_flavor:0 "#F Faerie Dragons are small, but highly effective at combating spellcasters. They share a close bond with both nature and the Night Elves, and are willing to fight in their defense."
+ farstriders_flavor:0 "#F The best rangers in Quel’thalas, the Farstriders are experienced, skilled, and noble. We could ask for no better guardians than them."
+ fel_guard_flavor:0 "#F The Felguard are our rank-and-file soldiers. They are as fearless as they are powerful, making them an impressive force worthy of leading the vanguard of our Crusade."
+ dark_ranger_flavor:0 "#F In life, they served Quel’thalas. In death, they serve us. Embittered but loyal, they serve as elite agents of our will, acting as saboteurs, archers, and generals as needed."
+ dragonhawk_rider_flavor:0 "#F In the wild they are clever predators, but when tamed, dragonhawks become noble steeds, allowing our soldiers to tear the enemy ranks apart with speed and precision."
+ golem_arcane_flavor:0 "#F Arcane golems are hulking, magically-powered constructs found mainly in territories ruled by the Kirin Tor. They act as peacekeepers and defenders, but when necessary they can be weapons of war."
+ golem_mech_flavor:0 "#F Mechanized golems are hulking, machine-powered robotic constructs. They act as peacekeepers and defenders, but when necessary they can be an unstoppable machine of war."
+ huntress_flavor:0 "#F Huntresses are the elite cadre of the night elf Sentinel army. These warrior women ride their loyal nightsaber companions into battle, mowing anyone down who stands in their way."
```

Comment:
While cutting on the other hand implies using only blade weapons, no spears of clubs or anything that can't "cut".

Reply:
In that case, mowing down could be fine I guess, but I would still change the word order so it's "mowing down anyone" instead of "mowing anyone down".